templates/docker-compose.template

```diff
@@ -48,15 +45,12 @@ services:
       - {{I2C_DEVICE}}:{{I2C_DEVICE}}
     restart: on-failure
     environment:
-      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
-      - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
```

Comment:
We should leave this in IMO

Reply:
The FIRMWARE_VERSION I mean

Reply:
can leave it but hm-gatewayrs is not using it.

Reply:
It's used to make sure the containers are removed and reinstalled at every update.

Otherwise Balena has issues and sometimes updates fail

Reply:
added back